### PR TITLE
Feat/home

### PR DESCRIPTION
**Pull request by MAfzalKhan1997**
Worked on Home Screen and Side NavBar

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -4,16 +4,98 @@ import seo from '../src/config/seo_en.json';
...
+  // HStack,
+  SimpleGrid,
+  Square,
+} from '@chakra-ui/react';
+// import Footer from '../src/components/Footer/Footer';
+import BaseImage from '../src/components/BaseImage/BaseImage';
 
 import MobileNav from '../src/components/Header/components/MobileNav';
 
+const quotesArray = [
```
Comment (mowsean) [inline pages/index.tsx:21]: I think there shouldn't be hard coded array we have a db call for this, also we can add image column in our db table as well.

File: pages/_app.tsx [hunk 2]
```diff
@@ -77,7 +80,19 @@ function MyApp(props: any) {
       </Head>
       <DefaultSeo {...seo.default} />
       <ChakraProvider theme={theme}>
-        <Component {...pageProps} />
+        <Flex>
```
Comment (mowsean) [inline pages/_app.tsx:83]: Is it necessary to use inline css here?

File: pages/index.tsx (hunk 1 above)
Comment (hussainanjar) [reply pages/index.tsx:21]: I agree, once we have API integration we can replace this array with API.

**Comment by hussainanjar** [comment]
@MAfzalKhan1997 please implement support for both EN and AR

File: pages/_app.tsx [hunk 3]
```diff
@@ -91,7 +94,19 @@ function MyApp(props: any) {
             defaultLocale={defaultLocale}
             messages={messages}
           >
-            <Component {...pageProps} />
+            <Flex>
```
Comment (umimehar) [inline pages/_app.tsx:97]: as discussed please move it to dashboard page with respective nested views

File: pages/index.tsx [hunk 4]
```diff
@@ -4,16 +4,98 @@ import seo from '../src/config/seo_en.json';
...
+
 const MainPage = () => {
   const {
     isOpen: mobileNavIsOpen,
     onClose: mobileNavOnClose,
     onOpen: mobileNavOnOpen,
   } = useDisclosure();
+  // style={{ backgroundColor: 'green' }}
+
+  const WelcomeSection = () => (
```
Comment (umimehar) [inline pages/index.tsx:60]: i love the approach you are creating smaller components.

File: src/components/Logo/Logo.tsx [hunk 5]
```diff
@@ -12,9 +12,7 @@ const Logo: FC<LogoProps> = (props: LogoProps) => {
         minH={'25px'}
         alt='Insurnace Market'
         {...props}
-        src={
-          'https://placeholder.com/wp-content/uploads/2018/10/placeholder.com-logo1.png'
-        }
+        src={'./assets/img/logo.png'}
```
Comment (umimehar) [inline src/components/Logo/Logo.tsx:15]: there is a logo component, try to reuse that. and we will make it a memo in react. :)

File: src/components/SideBar/SideBar.tsx [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+import BaseImage from '../BaseImage/BaseImage';
+
+const ROUTES = [
+  { title: 'Policies', path: '/' },
+  { title: 'Quotes', path: '/' },
+  { title: 'Claims', path: '/' },
+];
+
+const SideBar = () => {
```
Comment (umimehar) [inline src/components/SideBar/SideBar.tsx:13]: as discussed side bar will be a drawer. we can customize that with our needs. lets start drawer with always open drawer.

**Review by umimehar — COMMENTED** [review]
having small components is very good, 
the images you are using please make sure to compress them using https://imagecompressor.com/ or try to use svg as much as possible. later on we will keep svgs in a cdn. :)

**Pull request merged**